charts/pega/templates/_pega-deployment.tpl

```diff
@@ -113,7 +113,7 @@ spec:
           {{- if .node.cpuLimit }}
             cpu: "{{ .node.cpuLimit }}"
           {{- else }}
-            cpu: 2
+            cpu: 4
```

Comment:
values.yaml and values-large.yaml already specify a cpu limit of 4 for the stream tier.  It might make sense to remove these the cpuLimit specification for the stream tier to avoid confusion if the defaults are ever changed again.

Reply:
@pega-embate are you sure - might be looking at C*? [values.yaml](https://github.com/pegasystems/pega-helm-charts/blob/master/charts/pega/values.yaml#L163) [values-large.yaml](https://github.com/pegasystems/pega-helm-charts/blob/master/charts/pega/values.yaml#L163)

Reply:
I do agree we could remove the resource specification there and leave it as the defaults.

Reply:
> values.yaml and values-large.yaml already specify a cpu limit of 4 for the stream tier. It might make sense to remove these the cpuLimit specification for the stream tier to avoid confusion if the defaults are ever changed again.

I didn't find that it is set to cpulimit  4 in the mentioned files for pega nodes. Are you looking at 'cpu' cassandra?

Reply:
My mistake - cpuLimit is set to 2 for stream in values.yaml.  We should still be removing this so we pick up the new limit of 4.